frontend/js/tests/pins/PinnedRecordingCard.test.tsx

```diff
@@ -90,29 +90,12 @@ describe("PinnedRecordingCard", () => {
     });
   });
 
-  describe("handleError", () => {
-    it("calls newAlert", async () => {
-      const wrapper = mount<PinnedRecordingCard>(
-        <PinnedRecordingCard {...{ ...props, newAlert: jest.fn() }} />
-      );
-      const instance = wrapper.instance();
-
-      instance.handleError("error");
-
-      expect(instance.props.newAlert).toHaveBeenCalledTimes(1);
-      expect(instance.props.newAlert).toHaveBeenCalledWith(
-        "danger",
-        "Error",
-        "error"
-      );
-    });
-  });
 
   describe("unpinRecording", () => {
-    it("calls API, updates currentlyPinned in state, and calls newAlert", async () => {
+    it("calls API, updates currentlyPinned in state", async () => {
       const wrapper = mount<PinnedRecordingCard>(
         <GlobalAppContext.Provider value={globalProps}>
-          <PinnedRecordingCard {...{ ...props, newAlert: jest.fn() }} />
+          <PinnedRecordingCard {...{ ...props }} />
```

Comment:
While you're in here…
```suggestion
          <PinnedRecordingCard {...props} />
```